---
layout: page
title: About Me
---

**Payal Banerjee, M.S. Bioinformatics**

[**pbpayal42@gmail.com**](mailto:pbpayal42@gmail.com)                                



_Payal Banerjee is a bioinformatician with expertise in next generation sequencing data analysis like RNA sequencing, Single cell RNA sequencing, ChIP sequencing, microbiome as well as microarray analysis. She has experience in Cloud computing (AWS) and  also tested bioinformatics platforms like Dragen, DNA Nexus and Cavatica. She is acquainted with programming languages Shell, R and Python. Currently she is a Bioinformatics Analystt, at Children&#39;s National Medical Center, performing high throughput data analysis along with testing and optimizing pipelines for various experiments. Previous to this she was Bioinformatics Analyst at Digicon where she developed pipelines on an in-house Hadoop-optimized cloud platform developed by Digicon. Before that, she was a Bioinformatics Analyst for an NIH contract within the Big Data to Knowledge (BD2K) program to develop a Biomedical Standards Coordinating Center to enhance data sharing and lower the barriers to standards adoption by using a variety of tools to educate, train, and advocate for the use of standards in research. Also she was a part-time Bioinformatics Analyst at Northrop Grumman performing data management for the Immport database._

**Technical Expertise:**

| **SN** | **Area** | **Technique** |
| --- | --- | --- |
| 1. | Programming | Bash scripting, Python, R, |
| 2. | Next Generation Sequencing Pipelines | RNA Sequencing, DNA Sequencing, Microbiome, T-cell Receptor Sequencing, Chip Seq, Single Cell sequencing, Microarray |
| 3. | Cloud Resource NGS Platforms | DNA Nexus, Illumina Basepace (including DRAGEN), Galaxy, Cavatica, GenePattern |
| 4. | Cloud Computing | AWS EC2 and S3 |
| 5. | Operating Systems | Unix (Linux,Ubuntu, Centos, macOS) |
| 6. | Website Management | Github Pages , Relational Databases (RDMS), MySQL, HTML, PHP, Apache, Data Model Design, Drupal |



**Bioinformatics Analyst                 November 2017 – Current**

**Children&#39;s National Medical Center (Washington, DC)**

- Develop pipelines and genomic software in Shell, R/Bioconductor, Python, and for executing high throughput analysis pipelines such as RNASeq metagenomics, ChIPSeq, ExomeSeq, microarray  and Single Cell sequencing analysis using leading-edge genotyping technology tools, such as STAR, BWA, GATK, Samtools, Limma, EdgeR, DESeq2, Seurat, Qiime1 for genomics data analysis
- Apply functional, and disease annotation to sets of SNPs using ANNOVAR, SnpEff
- Conduct downstream data analysis along with data curation with IPA, Pathway Studio, GSEA
- Visualize complex datasets using graph-based softwares, such as Integrated Genome Viewer, UCSC Genome Browser, R, Cytoscape,
- Interpret and infer results from public databases such as NCBI, Ensembl, TCGA, cBioPortal, dbGaP, SRA
- Organize, analyze, and mine large-scale genomic data sets including integration with data available from public sources
- Run analysis jobs on Cloud platforms like AWS, Dragen, DNA Nexus, Cavatica for genomic data analysis and also Unix/Linux based system, in a cluster environment
- Organize and present educational and training presentations on next generation sequencing techniques and pipelines for the core workshops and seminars
- Work closely with research lab scientists, in support of a diverse set of research programs and aid in the process of experimental designs and applied statistics for study base grants. Also provide data support, bioinformatics, systems analysis, and research design and development



**Bioinformatics Specialist                    January 2017 – September 2017**

**Digicon Corp. (McLean, VA)**

- Testing Next Generation Sequencing pipelines on an in-house bioinformatics analysis platform
- Perform quality control and analysis of SNP for next-generation sequencing
- Apply functional, and disease annotation to sets of SNPs
- Organize, analyze, and mine large-scale DNA sequencing data.
- Perform educational and training presentations on Next Generation Sequencing techniques and pipelines
- Assist in Proposals and grant applications



**Research Analyst                                                                                          April 2017 – September 2017**

**Northrop Grumman Information Technology Health Solutions (Rockville, MD)**

**Client: National Institute of Allergy and Infectious Diseases (NIAID), NIH**

- Curation of relevant immunological data for pertinent datasets.
- Quality control and analysis of appropriate missing data upon public request
- Mapping relevant metadata for data harmonization and relevance.



**Bioinformatics Analyst                     December 2015 – October 2016**

**ESAC, Inc. (Rockville, MD)**

**Client: Big Data to Knowledge (BD2K) program, NIH (****October 2015 – Present)**

- Assemble, curate and maintain standards data and related entities, relevant to biomedical research as a part of the project.
- Perform systems analysis and serve as an expert in researching and recommending standards for use in various biomedical domains, especially in research.
- Develop data model and metadata schema for database implementation.
- Assist in GitHub maintenance as a data sharing drive undertaken by the project.
- Prepare standards team monthly reports, as required.
- Assist the team in writing Statement of Purpose (SOP), regarding various tasks pertaining to the standards team.

**Related Experience:**

**ICBI, Georgetown University**

**Summer Internship (2015)**

- Performed high-throughput systems biology analysis on Brain tumor clinical data using G-DOC Plus, R, Pathway Studio and other statistical tools.

**Department of Bioinformatics, Georgetown University**

- **Database Project (2015):** Created a database for proteins related to Diabetes using MySQL, XAMPP, PHP, HTML to assimilate all the pathway, sequence, PubMed Ids details of the specific proteins.
- **Python Project (2015):** Designed a web-server application using Python to carry out an in silico enzymatic digest of a user-provided protein sequence or Uniprot accession.
- **Little Skate Genome Annotation Project  (2014):** Characterization of coding region in the _Leucoraja erinacea_ by annotation of a skate gene from genomic data and transcriptomic data and also annotated an unreviewed elasmobranch protein.

**Education:**

| **SN** | **Degree** | **Field** | **University/College** | **Timeline** | **Place** |
| --- | --- | --- | --- | --- | --- |
| 1. | M.S. | Bioinformatics | Georgetown University | 2014-2015 | Washington, DC, USA |
| 2. | M.S. | Biochemistry | Bangalore University | 2011-2013 | Bangalore, India |
| 3. | B.S. | Biotechnology, Microbiology, Biochemistry | Bangalore University | 2008-2011 | Bangalore, India |

** **

**Publications**

- Immunotherapy of Relapsed and Refractory Solid Tumors With Ex Vivo Expanded Multi-Tumor Associated Antigen Specific Cytotoxic T Lymphocytes: A Phase I Study Journal of Clinical Oncology 2019 37:26, 2349-2359
- Enabling precision medicine in neonatology, an integrated repository for preterm birth research [Scientific Data](https://www.nature.com/sdata) volume 5, Article number: 180219 (2018)

### **Orchid ID -** https://orcid.org/0000-0003-1599-0679

**Workshop:**

- Internal Children&#39;s Bioinformatics Workshop Series
- ICSB DC RSG Summer 2016 Workshop, University of Maryland

Bacterial genome assembly &amp; annotation.

**Certifications:**

**Agile University**

- Agile Basics for Scrum Teams Course (2016)

**NOTE: I am on H1B (cap-exempt, non-profit institutions) and require sponsorship to work within USA.**
