---
layout: post
title: miRNA sequencing pipelines
---

1. mirdeep2 - https://www.mdc-berlin.de/content/mirdeep2-documentation
2. miRge 2.0 - https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-018-2287-y
3. Database 
  a. Mirbase - http://www.mirbase.org/
  b. RFAM - https://rfam.xfam.org/
  c. TargetScan - http://www.targetscan.org/vert_72/
4. Pathway - DIANA-mirPath is a miRNA pathway - http://snf-515788.vm.okeanos.grnet.gr/

Reference Theods sections:
1. https://www.nature.com/articles/s41598-019-53377-x#Sec11
'miRNA sequencing and statistical analysis

Eight small RNA libraries were prepared using the TruSeq® Small RNA Sample Prep Kit (Illumina®). All of the procedures and standards were performed as described in the instructions supplied with the kit. After quality control, eight libraries were sequenced on a HiSeq. 2000 platform (Illumina) and 50-bp single-end reads were obtained. Before reads were analysed, the adapters were removed and low-quality reads were discarded using the Fastx-toolkit for quality control48. The CAP-miRSeq pipeline was used for miRNA analysis49. MiRNA data were converted from fatsq to fasta format using awk in the Linux system. MirDeep2 was used to analyse miRNA sequencing data50. The miRNA reference was downloaded from the miRBase database4. The expression of microRNAs in different libraries was normalised by Trimmed Mean of M-values (TMM)51, and differentially expressed miRNAs (P-value ≤ 0.05, and FC ≥ 2 or ≤ 0.5, as |Log2 FC| ≥ 1) were identified using the edgeR package52.'
